software/base/src/main/java/org/apache/brooklyn/entity/software/base/AbstractSoftwareProcessDriver.java

```diff
@@ -189,7 +187,7 @@ public void start() {
      */
     public void preInstall() {}
 
-    public abstract void runPreInstallCommand(String command);
+    public abstract void runPreInstallCommand();
```

Comment:
Same here: the `runPreInstallCommand` should have the same signature as `runPostInstallCommand` etc.